test/suites/vm.sh

```diff
@@ -9,7 +9,9 @@ test_vm_empty() {
     return
   fi
 
-  if grep -qxF 'VERSION_ID="22.04"' /etc/os-release; then
+  isJammyRunner=$(grep -qxF 'VERSION_ID="22.04"' /etc/os-release && echo 0 || echo 1)
```

Comment:
This doesn't look right as jammy runners should match 22.04 and don't support virtiofsd